modules/finance/src/main/java/com/opengamma/strata/finance/rate/bond/FixedCouponBondTrade.java

```diff
@@ -456,14 +513,31 @@ public Builder quantity(long quantity) {
       return this;
     }
 
+    /**
+     * Sets the upfront fee payment of the bond trade.
+     * <p>
+     * The payment sign should be compatible with the product buy/sell flag, i.e. payment is negative for buy and
+     * positive for sell bond.
```

Comment:
Replace with: "The payment sign should be compatible with the product quantity, i.e. payment is negative for positive quantity and positive for negative quantity."